### DISCUSSION TITLE
Auth0 Passwordless

### DISCUSSION THREAD

**Discussion by miguel-a-calles-mba** [nuxt-community/auth-module#1596] [Support Requests]
Hi, is it possible to use Auth0 in passwordless mode? This module works great with a regular username and password login, but my use case requires passwordless login. I am not sure how to enable that.

**Comment by bmulholland** [comment]
We don't have support for this today, and I don't use auth0 so I don't know. If anyone figures it out, please post code or configuration to enable it and we can look at pulling it into the module.
